frontend/packages/ceph-storage-plugin/src/queries/ceph-queries.ts

```diff
@@ -55,12 +58,30 @@ export const DATA_RESILIENCY_QUERY = {
 export const UTILIZATION_QUERY = {
   [StorageDashboardQuery.CEPH_CAPACITY_USED]:
     'sum(kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"}))',
-  [StorageDashboardQuery.UTILIZATION_IOPS_QUERY]:
-    '(sum(rate(ceph_pool_wr[1m])) + sum(rate(ceph_pool_rd[1m])))',
-  [StorageDashboardQuery.UTILIZATION_LATENCY_QUERY]:
-    '(quantile(.95,(cluster:ceph_disk_latency:join_ceph_node_disk_irate1m)))',
-  [StorageDashboardQuery.UTILIZATION_THROUGHPUT_QUERY]:
-    '(sum(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])))',
+  [StorageDashboardQuery.UTILIZATION_IOPS_READ_QUERY]: {
+    query: 'ceil(sum(rate(ceph_pool_rd[1m])))',
```

Comment:
why use the `ceil()` function?

Reply:
Usually the scrape interval for prometheus is `15s`, but here the scrapping interval is set to `1m`.
@rexagod , if the thumb rule for range interval is 4 times the scrape interval, then we need to change it to 4m (for both read and write queries).

About the recording rule, as this change has to be done in a different project all together, I will request to do this in a separate PR.
`ceil()` function is used (or how I interpret it as) to show that if we have crossed the base value (like 2.0 is crossed to 2.1) we consider it to be in the bracket of next value (that is in this example 3.0).

Reply:
Made changes [here](https://github.com/openshift/console/pull/10470/files#diff-ebb08deedd0b9367aa8dab048c98e457d2d3ec14cd0f86756c29c73c095656acR61-R84). Resolving this.